### DISCUSSION TITLE
Do not see IP Ban in Incoming rules

### DISCUSSION THREAD

**Discussion by tko4315** [DigitalRuby/IPBan#319] [General]
Hi. I am running Server 2016 Datacenter and installed IPBan. However, I do not see IpBan in the incoming rule. I restarted server and still nothing.

Do I need to change another setting? Thanks!

**Comment by jjxtra** [comment]
IPBan Pro uses Windows Filtering Platform.

https://ipban.com/windows-filtering-platform/

**Reply by tko4315** [reply]
i am running the free version. Should IP Ban show in Incoming rules?

**Reply by tko4315** [reply]
I opened the Filters file and searched for IPBan_ but it did not find anything.

**Reply by tko4315** [reply]
I installed pro. Thanks
